### DISCUSSION TITLE
Standalone topic operator deployment fails

### DISCUSSION THREAD

**Discussion by ss85705** [strimzi/strimzi-kafka-operator#8540] [Q&A]
@scholzj
I am trying to deploy strimzi standalone topic and user operator deployment which is supposed to connect with Kraft enabled Kafka cluster(not managed by Strimzi). 
I just updated the STRIMZI_KAFKA_BOOTSTRAP_SERVERS with running kafka address and removed the STRIMZI_ZOOKEEPER_CONNECT and other ZK related variables. 
I also added STRIMZI_KRAFT_ENABLED = true 

After doing above changes, the deployment is not showing up. 

strimzi-topic-operator     0/1     0            0           7m52s
strimzi-user-operator      0/1     0            0           50s

Can you please help.

**Comment by scholzj** [comment]
The current version of the Topic operator requires (and will always require) access to ZooKeeper because Kafka does not provide parity features. You can check the Strimzi Proposal number 53 which describes a new version of the Topic Operator which will be compatible also with KRaft brokers: https://github.com/strimzi/proposals/blob/main/051-unidirectional-topic-operator.md
